### PR TITLE
[MIG] account_invoice_line_report: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
- Update manifest
- Update views
- Update translation

cc @Tecnativa

### CODE REVIEW

File: account_invoice_line_report/i18n/account_invoice_line_report.pot [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline account_invoice_line_report/i18n/account_invoice_line_report.pot]: Remove POT file.

File: account_invoice_line_report/i18n/antic.po [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline account_invoice_line_report/i18n/antic.po]: Why this language name?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by feketemihai** [comment]
@pedrobaeza Do you use this one, why not using the invoice reporting because it's already based on the invoice lines, just you don't have the filter without price? Is it mandatory in your workflow?

**Comment by pedrobaeza** [comment]
Yes, it's not enough for making some aggregation and it's simpler to have the list view in this case. We have several customers using this directly.

**Review by feketemihai — APPROVED** [review]
Ok... Thanks for the feedback.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**